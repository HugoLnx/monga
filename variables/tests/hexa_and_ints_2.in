int main() {
  int x; 
  x = 0x542;
  x = x + 3;
}
