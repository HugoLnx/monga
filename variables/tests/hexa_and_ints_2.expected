declarations:
  declaration:
    function:main,259,0
      block:
        var_declarations:
          variables:
            variable:x,259,0
        statements:
          statement:
            attribution:
              var:x
              exp:1,0x542
          statement:
            attribution:
              var:x
              exp:7
              binary:0
                exp:5
                  var:x
                exp:3,3.00000